src/components/ItemModalBackground.tsx

```diff
@@ -0,0 +1,201 @@
+                flexDirection: "row",
+                alignItems: "center",
+                paddingBottom: 20,
+                paddingTop: 20,
+              }}
+            >
+              <div style={{ marginRight: 10 }}>
+                <h3>Select Tags</h3>
+              </div>
+              <div>
+                <button
+                  type="button"
+                  className={selectedTags.length > 0 ? "tag" : "tagSelected"}
+                >
+                  {selectedTags.length} selected
+                </button>
+              </div>
+            </div>
+
+            {/* Category Selectable Buttons */}
+            <div
+              style={{
+                display: "flex",
+                flexDirection: "row",
+                alignItems: "center",
+                paddingBottom: 30,
+              }}
+            >
+              <button
+                style={{ marginRight: 10 }}
+                type="button"
+                className={
+                  selectedTags.includes("Clothing") ? "tag" : "tagSelected"
+                }
+                onClick={() => handleTagClick("Clothing")}
+              >
+                Clothing
+              </button>
+              <button
+                style={{ marginRight: 10 }}
+                type="button"
+                className={
+                  selectedTags.includes("Collectibles") ? "tag" : "tagSelected"
+                }
+                onClick={() => handleTagClick("Collectibles")}
+              >
+                Collectibles
+              </button>
+              <button
+                style={{ marginRight: 10 }}
+                type="button"
+                className={
+                  selectedTags.includes("Jewlery") ? "tag" : "tagSelected"
+                }
+                onClick={() => handleTagClick("Jewlery")}
+              >
+                Jewlery
+              </button>
+              <button
+                style={{ marginRight: 10 }}
+                type="button"
+                className={selectedTags.includes("Art") ? "tag" : "tagSelected"}
+                onClick={() => handleTagClick("Art")}
+              >
+                Art
+              </button>
+              <button
+                style={{ marginRight: 10, width: 150 }}
+                type="button"
+                className={
+                  selectedTags.includes("Household Items")
+                    ? "tag"
+                    : "tagSelected"
+                }
+                onClick={() => handleTagClick("Household Items")}
+              >
+                Household Items
+              </button>
+            </div>
+          </div>
+
+          {/* Submit Button */}
+          <div style={{ display: "flex", justifyContent: "center" }}>
+            <button
+              style={{ borderRadius: 10 }}
+              type="button"
+              className="dropbtn"
+              onClick={() => console.log(formData)}
```

Comment:
Create a handleSubmit function that validates the data and closes the modal. Name shouldn't be empty. Price should be a valid dollar amount.